eodag/plugins/search/data_request_search.py

```diff
@@ -113,6 +113,7 @@ def _create_data_request(self, product_type, eodag_product_type, **kwargs):
                     str(e),
                     request_job.text,
                 )
+                raise
```

Comment:
Use `eodag.utils.exceptions.RequestError` instead of `logger.error() ; raise`

Reply:
Done.